app/modules/ads/module.js

```diff
@@ -61,6 +62,8 @@ class Ads {
     events.on(events.PAGE_CHANGE_EVENT, this.callBidders);
     this.callBidders();
 
+    this.configureCheshireCat(instantGlobals);
```

Comment:
s/configureCheshireCat/configureBillTheLizard ?
Rebase with dev please. Should we configure BTL before labrador tracking so it will contain model variables?

Reply:
Renamed. I will create a follow up ticket for tracking - this ticket is already quite big.